src/components/iconButton/iconButton.tsx

```diff
@@ -13,6 +13,10 @@ import MATIC from '../../assets/MATIC.png';
 import XTZ from '../../assets/XTZ.png';
 import AVAXC from '../../assets/AVAXC.png';
 import SEGWIT from '../../assets/SEGWIT.png';
```

Comment:
You may wonder why the token and network names are in all caps ... since we got the icons from Daniel I didn't want you to rename the files ... theoretically `import SEGWIT from '../../assets/SEGWIT.png';` should become `import SEGWIT from '../../assets/segwit.png';`

If you want feel free to adjust those file imports and the names - but you just if you are up for it :)

Reply:
the problem is that i can not import anything by name 'default' so it was my solution
If we need to rename, i will add 'info' instead of 'default' and add some colors for this toasts. :)